bots/notify/src/main/java/org/openjdk/skara/bots/notify/IssueUpdater.java

```diff
@@ -216,6 +218,20 @@ private Issue findIssue(Issue primary, String fixVersion) {
         return createBackportIssue(primary);
     }
 
+    private Optional<HostUser> findIssueUser(Commit commit) {
+        var authorEmail = EmailAddress.from(commit.author().email());
+        if (authorEmail.domain().equals("openjdk.org")) {
+            return Optional.of(issueProject.issueTracker().user(authorEmail.localPart()));
+        } else {
+            var committerEmail = EmailAddress.from(commit.committer().email());
+            if (!committerEmail.domain().equals("openjdk.org")) {
+                log.severe("Cannot determine issue tracker user name from committer email: " + committerEmail);
+                return Optional.empty();
+            }
+            return Optional.of(issueProject.issueTracker().user(committerEmail.localPart()));
+        }
+    }
```

Comment:
Instead of repeating `Optional.of(issueProject.issueTracker().user(...))`, you might want to code this like:
```
        String username = null;

        var authorEmail = EmailAddress.from(commit.author().email());
        if (authorEmail.domain().equals("openjdk.org")) {
            username = authorEmail.localPart();
        } else {
            var committerEmail = EmailAddress.from(commit.committer().email());
            if (!committerEmail.domain().equals("openjdk.org")) {
                log.severe("Cannot determine issue tracker user name from committer email: " + committerEmail);
            }
            username = committerEmail.localPart();
        }

        return username == null ? Optional.empty() : Optional.of(issueProject.issueTracker().user(username));
```

Reply:
Oh, I should mention that I'm fine your version as well :+1: If you want to use your version with early return, then please remove the `else {` part after the return :smiley:

Reply:
Fair enough, how about this? :)